### PR TITLE
Shane/expose pattern variables

### PR DESCRIPTION
**Pull request by LivingSynthesis**

### CODE REVIEW

File: VortexEngine/src/patterns/multi/DripMorphPattern.cpp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #include "DripMorphPattern.h"
 
+// NOTE: I'm not sure this is right but I'm replicating the recent changes to Blinkstep here
```
Comment (Unreal-Dan) [inline VortexEngine/src/patterns/multi/DripMorphPattern.cpp:3]: Can dripmorph be written as a BlinkStep pattern? if not then this is fine

**Review by Unreal-Dan — COMMENTED** [review]
looks good just remove the note comment

File: VortexEngine/src/patterns/multi/DripMorphPattern.cpp (hunk 1 above)
Comment (LivingSynthesis) [reply VortexEngine/src/patterns/multi/DripMorphPattern.cpp:3]: I feel like it can be, should I convert it to a blinkstep pattern?

Comment (Unreal-Dan) [reply VortexEngine/src/patterns/multi/DripMorphPattern.cpp:3]: I don't think it can, I think that's why we didn't do it.

Comment (Unreal-Dan) [reply VortexEngine/src/patterns/multi/DripMorphPattern.cpp:3]: feel free to try, doesn't matter

Comment (LivingSynthesis) [reply VortexEngine/src/patterns/multi/DripMorphPattern.cpp:3]: meh XD if it works it works i guess

**Review by Unreal-Dan — APPROVED** [review]

**Pull request merged**